test/JSON/jsonerrorbuffer.js

```diff
@@ -0,0 +1,11 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+let desiredLength = 10 * 1000 * 1000;
+//desiredLength -= 1;
+const json = `"${'a'.repeat(desiredLength - 3)}",`; // "aaa...aaa",
+print(json.length);
+JSON.parse(json);

```

Comment:
Instead of printing the whole error - could we just try/catch and print('pass') on that? you could avoid baseline... also I am not sure as the baseline contains the hard path.

Reply:
good point